wallet/Cargo.toml

```diff
@@ -8,6 +8,7 @@ repository = "https://github.com/grease-xmr/grease"
 
 [dependencies]
 libgrease = { version = "0.1.0", workspace = true }
+circuits = { workspace = true }
```

Comment:
It looks like this can move to `dev-dependencies`

Reply:
The structure of `wallet/examples` prevents this, but those could be made into tests.